src/command.cpp

```diff
@@ -523,7 +529,13 @@ void Command::show(uuid::console::Shell & shell, uint8_t device_type, bool verbo
     std::list<std::string> sorted_cmds;
     for (const auto & cf : cmdfunctions_) {
         if ((cf.device_type_ == device_type) && !cf.has_flags(CommandFlag::HIDDEN)) {
-            sorted_cmds.push_back(tagged_cmd(cf.cmd_, cf.flags_));
+            // remove duplicates
+            auto line = tagged_cmd(cf.cmd_, cf.flags_);
+            // only add if not already in list
+            // removes duplicates if having more than one device (e.g. 2 thermostats)
```

Comment:
This is not needed. We don't have dublicates, they always differ in heatingcircuit.

Reply:
yes you're right - i will need to find why in the tests when I add to thermostats with two different HCs its not working (the test is called 2thermostats)

Reply:
and I assume having multiple solar, mixers etc won't happen either? just to double check